pytorch_lightning/trainer/states.py

```diff
@@ -36,6 +37,17 @@ class TrainerState(str, Enum):
     INTERRUPTED = 'INTERRUPTED'
 
 
+class TrainPhase(LightningEnum):
```

Comment:
I don't like the name. `testing` is not a training phase...

Call it `RunningStage`?